server/src/main/java/com/linecorp/centraldogma/server/CentralDogmaConfig.java

```diff
@@ -75,6 +81,42 @@
  */
 public final class CentralDogmaConfig {
 
+    private static final Logger logger = LoggerFactory.getLogger(CentralDogmaConfig.class);
+
+    private static final List<ConfigValueConverter> CONFIG_VALUE_CONVERTERS;
+
+    static {
+        final ArrayList<ConfigValueConverter> configValueConverters = new ArrayList<>();
+        Streams.stream(ServiceLoader.load(ConfigValueConverter.class)).forEach(configValueConverters::add);
+        configValueConverters.add(DefaultConfigValueConverter.INSTANCE);
+        CONFIG_VALUE_CONVERTERS = ImmutableList.copyOf(configValueConverters);
+        logger.debug("Available {}s: {}", ConfigValueConverter.class.getName(), CONFIG_VALUE_CONVERTERS);
+    }
```

Comment:
- Can we keep a `Map<String /* prefix */, ConfigValueConverter>` and fail when duplicate prefixes are found?
- We could then log a `Map<String /* prefix */, String /* class name *>` instead of ugly `className@12345`.

Reply:
That's a good suggestion. All addressed. 👍